trans_sec/analytics/oinc.py

```diff
@@ -355,7 +355,7 @@ def __process(self, int_data):
                     'Not calling SDN as last attack notification for %s'
                     ' was only %s seconds ago',
                     attack_dict, time.time() - last_attack)
-                return False
+                return True
```

Comment:
Why was this changed to True now? Doesn't that break stuff upstream?

Reply:
@spisarski  With the _test_start_two_attacks_ we send two packets - the first packet detects an attack and ret_val1 is set to True and the second packet also detects an attack but just doesn't send the SDN controller the notification (because the notification was sent by packet 1) and so ret_val2 was being set to False. The test case _test_start_two_attacks_ was failing because ret_val1=True and ret_val2=False did not apply to any of the 'Expecting '  conditions. It seemed like a bug so changed it to True if an attack is detected irrespective of if the notification is being sent or not.